lib/ansible/modules/windows/win_uri.py

```diff
@@ -28,7 +28,6 @@
     description:
     - The HTTP Method of the request or response.
     type: str
-    choices: [ CONNECT, DELETE, GET, HEAD, MERGE, OPTIONS, PATCH, POST, PUT, REFRESH, TRACE ]
```

Comment:
i would add same note as for the other module about the restriction being lifted

Reply:
Do we have to ? We are only creating more wall of text for something that's no longer an issue.

PS The plan is to backport this to v2.7 and hopefully v2.6 as well.